domain/domain.py

```diff
@@ -242,7 +259,7 @@ def get_domain_cell(self, attr, row):
 
         This would produce [B,C,E] as domain values.
 
-        :return: (initial value of entity-attribute, domain values for entity-attribute).
+        :return: (list of initial values, current value, list of domain values).
         """
 
         domain = set([])
```

Comment:
can we make this `set()`? no need to initialize it from an empty list

Reply:
Done.